File: 04_loops/c/loop_until_ok.c
#include <stdio.h>
#include <string.h>

int main(void) {
    char input[100];

    while (1) {
        printf("Type ok: ");
        scanf("%99s", input);

        if (strcmp(input, "ok") == 0) {
            break;
        }

        printf("Try again.\n");
    }

    printf("Finished.\n");

    return 0;
}

File: 04_loops/go/loop_until_ok.go
package main

import "fmt"

func main() {
	var input string

	for {
		fmt.Print("Type ok: ")
		fmt.Scanln(&input)

		if input == "ok" {
			fmt.Println("OK!")
			break
		}

		fmt.Println("Try again.")
	}
}
